lib/android/examples/src/ui_test.nit

```diff
@@ -20,44 +20,37 @@ module ui_test is
 	app_version(0, 1, git_revision)
 	app_namespace "org.nitlanguage.ui_test"
 	android_manifest_activity """android:theme="@android:style/Theme.Light""""
```

Comment:
For the sake of space (and the colouring on Github), maybe the `"""` could be simple `"` and escaping the two instances of `"` inside of the string would be a breeze for review :)